Protocol.md

```diff
@@ -81,6 +82,11 @@ the funding transaction and CETs.
    * [`u64`:`feerate_per_vb`]
    * [`u32`:`cet_locktime`]
    * [`u32`:`refund_locktime`]
+   * [`offer_tlvs`: `tlvs`]
```

Comment:
I take it none of `offer_tlvs`, `accept_tlvs` or `sign_tlvs` have any currently existing fields correct?

Reply:
correct